awx/settings/defaults.py

```diff
@@ -449,7 +449,6 @@ def IS_TESTING(argv=None):
     'gather_analytics': {'task': 'awx.main.tasks.system.gather_analytics', 'schedule': timedelta(minutes=5)},
     'task_manager': {'task': 'awx.main.scheduler.tasks.task_manager', 'schedule': timedelta(seconds=20), 'options': {'expires': 20}},
     'dependency_manager': {'task': 'awx.main.scheduler.tasks.dependency_manager', 'schedule': timedelta(seconds=20), 'options': {'expires': 20}},
-    'workflow_manager': {'task': 'awx.main.scheduler.tasks.workflow_manager', 'schedule': timedelta(seconds=20), 'options': {'expires': 20}},
```

Comment:
I think we can get rid of this periodic scheduler - the task manager will schedule a workflow manager to run if it sees a running workflow job

Reply:
Yes, I was going to suggest doing a version of exactly this, but you did it even better than what I was thinking. Since the queryset of WorkflowManager is exactly the running workflow jobs, we're pretty safely covered.